prism/nbconverted/0.download_prism.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Download PRISM Repurposing Data
+# 
+# https://depmap.org/repurposing/
```

Comment:
I love that you're providing links to the data sources (here and elsewhere). Consider also including a [`CITATION.cff` file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) with relevant references to this repo for the data sources. Doing this might make it easier to both credit and understand what sources were used.

Reply:
my understanding of these CITATION files is for folks to learn how to cite _the repository_ and not to cite external projects